Sources/Utilities.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+struct Utilities {
```

Comment:
Ironically 😛 I just moved those from string extensions to a new class. I had some problem before when some frameworks have the same method on String